CMakeLists.txt

```
@@ -1,5 +1,8 @@
 cmake_minimum_required(VERSION 3.22.1)
 
+list(APPEND CMAKE_MODULE_PATH ${CMAKE_CURRENT_LIST_DIR}/CMake)
```

Lets rename this directory, so it doesn't look like CMake itself is one of our dependencies.
(also I added quotes to the path)

```suggestion
list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_LIST_DIR}/cmake-settings")
```

---

This is a different `CMake` that contains `ystdlib-cpp-helpers`